arrow-ipc/src/convert.rs

```diff
@@ -31,17 +31,62 @@ use crate::writer::DictionaryTracker;
 use crate::{size_prefixed_root_as_message, KeyValue, Message, CONTINUATION_MARKER};
 use DataType::*;
 
-pub struct IpcSchemaConverter<'a> {
-    dictionary_tracker: &'a mut DictionaryTracker,
+/// Low level Arrow [Schema] to IPC bytes converter
+///
+/// See also [`fb_to_schema`] for the reverse operation
+///
+/// # Example
```

Comment:
I always found these APIs confusing to use, so I figured I would make them easier to use here